website/docs/other-frameworks.mdx

```diff
@@ -102,15 +102,63 @@ $result = GraphQL::executeQuery($schema, $query, null, new Context(), $variableV
 
 The smallest working example using no framework is:
 
-```php
+```json title="composer.json"
+{
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/"
+    }
+  },
+  "require": {
+    "thecodingmachine/graphqlite": "^6",
```

Comment:
I think we can use an asterisk, `*` for the version here.  We're now at 7.0 - no need to specify.